applications/crossbar/priv/api/swagger.json

```diff
@@ -12702,37 +12702,9 @@
         "version": "2.0"
     },
     "paths": {
-        "/v2/about": {
-            "get": {}
-        },
```

Comment:
I like the attributes idea. 
But what if a module provides both a root and masqueradable API?

What is the mechanism that enables or disables prepending accounts in path?
Ah I remember: it's s matter of whether account_id is defined in Context. 

Maybe we can add an explicit way of creating APIs to root?
Something like allowed_root_methods(). 
That would be parsable but requires updating all cb_*

> On 29 Jul 2016, at 18:23, lazedo <notifications@github.com> wrote:
> 
> In applications/crossbar/priv/api/swagger.json:
> 
> > @@ -12702,37 +12702,9 @@
> >          "version": "2.0"
> >      },
> >      "paths": {
> > -        "/v2/about": {
> > -            "get": {}
> > -        },
> @jamesaimonetti yep, we know that.
> how about using module attributes to mark these ? something like
> 
> -api(root).
> and then we could use Module:module_info(attributes) and look for api ?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>